src/github.com/getlantern/go-natty/natty/natty.go

```diff
@@ -37,7 +37,11 @@ var (
 func init() {
 	nattyBytes, err := bin.Asset("natty")
 	if err != nil {
```

Comment:
We're logging instead of panicking to allow Android hosts (where natty is disabled by build tags) to continue working.